src/lsp.rs

```diff
@@ -19,14 +19,15 @@ pub struct Quote {
 pub async fn request_quote(
     p2p_address: &P2PAddress,
     refund_address: &Address,
+    lsp_endpoint: &String,
```

Comment:
The point of this PR IMO is to make the LSP stubbable or testable. That means initializing it from a config dependency that can be changed on test. I don't think the endpoint should be a parameter of this function unless it's attached to `self`